src/main/java/com/qunite/api/security/JwtService.java

```diff
@@ -4,31 +4,48 @@
 import com.auth0.jwt.algorithms.Algorithm;
 import com.auth0.jwt.interfaces.DecodedJWT;
 import com.qunite.api.domain.User;
+import com.qunite.api.service.UserService;
 import java.time.Instant;
 import java.util.Optional;
+import lombok.RequiredArgsConstructor;
 import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Lazy;
 import org.springframework.stereotype.Component;
 
 @Component
+@RequiredArgsConstructor
 public class JwtService {
   @Value("${jwt.secret}")
```

Comment:
use @RequiredArgsConstructor(onConstructor = @__(@Lazy))

Reply:
According to Lombok API docs, the recommended way is onConstructor_ ={@Lazy} for JDK8+. For JDK7 is onConstructor = @__(@Lazy)

source: https://stackoverflow.com/a/70519537/20311078

Reply:
I don`t like annotating all dependencies with lazy, it`s better to move lombok config to src and fix the problem.

Reply:
> I don`t like annotating all dependencies with lazy, it`s better to move lombok config to src and fix the problem.

@RequiredArgsConstructor creates constructor only on FINAL fileds. It means String fields will not be affected and it seems better to use already used annotation rather than separate config

Reply:
`@RequiredArgsConstructor` creates constructor only on FINAL fileds. It means String fields will not be affected and it seems better to use already used annotation rather than separate config

Reply:
Final fields is still almost all Fields, i don't like the idea of lazining them without a reason